scripts/dump_load_db.sh

```diff
@@ -1,92 +1,113 @@
-#!/bin/bash -e
-
-cd $(dirname $0)
-
-# some config variables
-function path {
-  if command -v realpath >/dev/null
-  then
-    realpath $1
-  else
-    echo "$1"
-  fi
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir=$( dirname "$0" )
+
+ENV_FILE="$scriptdir/../.docker/database-variables.env"
+CONTAINER='herdbook-db'
+
+realpath () {
+	if command -v realpath >/dev/null; then
+		command realpath --relative-to="$PWD" "$1"
+	else
+		printf '%s\n' "$1"
+	fi
 }
 
-OUTPUT_DIR="$(path ${2:-../db_dumps})"
-BASE="db-dump"
-TIMESTAMP=$(date +"%Y-%m-%d_%H%M")
-ENV_FILE=../.docker/database-variables.env
-CONTAINER="herdbook-db"
-DUMP_FORMAT="tar"
+usage () {
+	cat <<-HELP
+	USAGE: $0 load [--clean] filename | dump [directory]
 
-if [[ ! "$1" =~ ^(load|dump)$ ]]
-then
-  cat << HELP
-USAGE: ./dump_load_db.sh [load [filename] | dump [directory]]
+	Sub-commands include
 
-Viable options are:
+	        load [--clean] filename
+	                                Load database dump from "filename".
+	                                The optional option "--clean" will drop
+	                                database objects before recreating them.
 
-      load <file> --clean       load database dump <file>. Optional flag --clean
-                                will drop database objects before recreating them
+	        dump [directory]
+	                                Create a dump file in "directory".
+	                                If no directory is given as an argument,
+	                                the dump file will be placed in
+	                                "$(realpath "$scriptdir/../db_dumps/")".
 
-      dump <directory>          create a dump file. If a directory is given as an
-                                argument the dump file will be placed there, else
-                                in projectroot/db_dumps/
+	         help
+	                                Show this help text.
+	HELP
 
-HELP
-  exit 0
-fi
+	exit 1
```

Comment:
There is an `exit 1` after calling for incorrect usage (and when provided because it's asked for, it makes more sense to exit with 0).

Reply:
Actually, since I put the help text into its own function, I think it makes more sense to exit from the caller instead than from the function.  Ok?  This `exit` is more or less a forgotten thing I was supposed to remove anyway.